source/javascripts/app/_lang.js

```diff
@@ -143,11 +143,8 @@ under the License.
   // returns a new query string with the new language in it
   function generateNewQueryString(language) {
     var url = parseURL(location.search);
-    if (url.language) {
```

Comment:
Weird. Any idea why this was here before? Seems unnecessary. It basically says, if there was previously a language, keep the whole url.search intact, but otherwise replace it all with the new language.